tests/numerics/ad/test_equation_system.py

```diff
@@ -60,6 +60,8 @@ def test_evaluate_variables():
             name=var_name, values=vals_sol, data=d, time_step_index=0
         )
         pp.set_solution_values(name=var_name, values=vals_it, data=d, iterate_index=0)
```

Comment:
I don't know how this test passed earlier, since no values were set for this iterate index?

Reply:
Yes it would crash, because it can't find values for `var.previous_iteration()`
But it did not crash before that... My guess there was a logic missing when accessing data with `prev_iter=True` but I could not find that part.